app/webpacker/stylesheets/_environments.scss

```diff
@@ -2,17 +2,22 @@
   background: govuk-colour("dark-grey");
 }
 
-.app-tag--staging,
 .app-tag--qa {
-  background: govuk-colour("yellow");
-  color: $govuk-text-colour;
+  background: govuk-colour("orange");
```

Comment:
Of course, `govuk` is going to spell `colour` correctly. :) Vive l'Alliance Anglaise!